### DISCUSSION TITLE
Sonar analyzer running on build after updating Refitter.SourceGenerator to 1.0.0

### DISCUSSION THREAD

**Discussion by Jappinen** [christianhelle/refitter#393] [General]
Hi,

I'm not sure if this should be a bug report or not but it strikes me very odd that after updating the Refitter.SourceGenerator package reference from 0.9.9 to 1.0.0 on my project I'm getting SonarAnalyzer warnings from my code on build output.

Is is this the intended behaviour?

**Comment by christianhelle** [comment]
@Jappinen Thanks for bringing this up. 

No, it is not my intention that the output source generator has SonarAnalyzer as a transitive dependency. I will address this immediately

**Comment by christianhelle** [comment]
[SonarAnalyzer.CSharp](https://www.nuget.org/packages/SonarAnalyzer.CSharp/) transitive dependency is removed in #394

**Comment by Jappinen** [comment]
Great, thanks! 1.0.1 works as expected.
